docs/md/manual/15_Interactive_services/15.4._Expose_node_filesystem.md

```diff
@@ -0,0 +1,68 @@
+# 15.4. Expose node filesystem
+
+- [Upload files through node's file browser](#Upload-files-through-node's-file-browser)
```

Comment:
For correct work of anchor-links in `Markdown` documents - link addresses should be written in the lowercase, spaces should be replaced by '-' symbol, other auxiliary symbols are also replaced by '-' or removed
```suggestion
- [Upload files through node's file browser](#upload-files-through-nodes-file-browser)
```
Analogically correct, please, other content items.